django/test/utils.py

```diff
@@ -113,21 +113,25 @@ def setup_test_environment(debug=None):
     if debug is None:
         debug = settings.DEBUG
 
-    _SavedSettings.debug = settings.DEBUG
-    settings.DEBUG = debug
+    # Storing previous values in the settings module itself is problematic.
+    # See #20636.  We store them in a private class instead.  Also, we use
+    # a Namespace object for convenience because it creates new attributes
+    # on assignment.
+    saved_data = Namespace()
```

Comment:
I couldn't find any special notation for this so something like `# TODO: ...` would probably work :) Updating the list in https://code.djangoproject.com/ticket/23919 would work too.